src/core/a-entity.js

```diff
@@ -264,15 +270,15 @@ var proto = Object.create(ANode.prototype, {
       if (isDependency && !isComponentDefined) {
         // Add component if it is a dependency and not yet defined.
         this.setAttribute(name, '');
-      } else {
-        if (this.isScene && !this.hasAttribute(name) && name in this.defaultComponents) {
-          // For scene default components, expose them in the DOM.
-          HTMLElement.prototype.setAttribute.call(this, name, this.defaultComponents[name]);
-        }
-        this.components[name] = new components[name].Component(this);
-        if (!this.paused) { this.components[name].play(); }
       }
 
+      if (this.isScene && !this.hasAttribute(name) && name in this.defaultComponents) {
```

Comment:
We need to understand why or it will cause problems down the road. Programming by coincidence is bad.